department-of-veterans-affairs/caseflow
# Added Bulk Building Without Rooms Logic

Resolves https://vajira.max.gov/browse/CASEFLOW-409

### Description
Added logic to the generate hearing days algorithm to add additional virtual days based on request in the RO assignments Excel doc. This involved creating a branch in the algorithm to ignore room distributions for virtual-only days

### Acceptance Criteria

### Testing Plan
1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Click the "Build Schedule" button at the top right
3. Select the RO/CO hearings from the dropdown
4. Input "01/01/2018" for the start date and "06/01/2018" for the end date and use [ROAssignmentTemplate.xlsx](https://github.com/department-of-veterans-affairs/caseflow/files/5784383/ROAssignmentTemplate.xlsx) to test
5. Click the "Continue" button and make sure that the algorithm runs successfully
6. Click the "Confirm assignments" and then the "Confirm upload" button when the modal pops up
7. Ensure you receive a success banner indicating the hearing days were created

### User Facing Changes

 BEFORE|AFTER
 ---|---
N/A |<img width="1789" alt="AfterAddColumn" src="https://user-images.githubusercontent.com/61989526/103955000-3370ab00-50fa-11eb-8381-ee875c3eb62f.png">


### Code Documentation Updates
